index.php

```diff
@@ -1,17 +1,33 @@
 <?php
 //Connect to Database:
 
+/**
+ * Connects to the database
+ *
+ * @return PDO
+ */
+function getDbConnection(): PDO {
```

Comment:
The three functions you've created are spot on, well done.

Now that you have them, can you move them into a new PHP file (just called functions.php)  which then gets required into this file - purely for organisational purposes! 

It means you could require the functions.php file into any other file to get access to the functions